js/ui/drop_down_button.js

```diff
@@ -331,7 +331,7 @@ const DropDownButton = Widget.inherit({
                     y: -1
                 }
             }
-        }, this._options.cache('dropDownOptions'));
+        }, this._options.cache('dropDownOptions'), { visible: this.option('opened') });
```

Comment:
Is this change necessary? What test covers this?

Reply:
https://github.com/DevExpress/DevExtreme/pull/11847/files#diff-0360e44628ce8bae07db8d6360fc7ba4R103

Reply:
Do we need to do this for `deferRendering` as well?

Reply:
No, `deferRendering` works correct, because `dropDownOptions.deferRendering` doesn't affect on popup:
- `deferRendering` is true: because the popup widget has not been created yet. Option will affect only on popup init.
- `deferRendering` is false: because the popup widget has already been created